src/Generators/MigrationGenerator.php

```diff
@@ -69,9 +69,9 @@ protected function buildDefinition(Model $model)
         /** @var \Blueprint\Models\Column $column */
         foreach ($model->columns() as $column) {
             $dataType = $column->dataType();
-            if ($column->name() === 'id' && $dataType === 'id') {
+            if ($column->name() === 'id' && $dataType === 'id' && $model->usesPrimaryKey()) {
                 $dataType = 'bigIncrements';
-            } elseif ($dataType === 'id') {
+            } elseif ($dataType === 'id' && $model->usesPrimaryKey()) {
```

Comment:
Why does this line need the check? Shouldn't it only be for disabling the `id` column?

Reply:
I misunderstood this line in the code.